bin/btmon.py

```diff
@@ -4713,6 +4798,11 @@ def process_calculated(self, packets):
                       options.mqtt_tls,
                       options.mqtt_map or MQTT_MAP,
                       options.mqtt_upload_period or MQTT_UPLOAD_PERIOD))
+    if options.sumo_out:
+      procs.append(SumoLogicProcessor
+                   (options.sumo_url or OEM_URL,
```

Comment:
This is incorrect, if sumo_url isn't set, it'll send it to openenergymonitor instead (and presumably fail)

Reply:
HUGE fail - thank you.